src/http/ngx_http_wasm_api.c

```diff
@@ -632,6 +643,81 @@ static ngx_str_t *ngx_http_wasm_get_scheme(ngx_http_request_t *r)
 }
 
 
+static ngx_str_t *
+ngx_http_wasm_get_status(ngx_http_request_t *r)
+{
+    ngx_str_t  *sc;
+    ngx_uint_t  status;
+    u_char     *p;
+
+    if (r->err_status) {
+        status = r->err_status;
+    } else {
+        status = r->headers_out.status;
+    }
+
+    sc = ngx_palloc(r->pool, sizeof(ngx_str_t));
```

Comment:
We can use some static variables to avoid allocation, like:
```
static ngx_str_t scheme_https = ngx_string("https");
```

Reply:
ok, thanks for the pointer.